app/client-app/src/app/pages/Wordlist/index.jsx

```diff
@@ -1,19 +1,32 @@
 import { Box } from "@chakra-ui/react";
 import { WordCardList } from "./components/WordCardList";
 import LightHeading from "components/core/LightHeading";
+import adjectives from "services/adjectives";
+import animals from "services/animals";
+import { fetchWordList } from "api/wordlist";
+import { useEffect, useState } from "react";
 
 const Wordlist = () => {
+  const [wordList, setWordList] = useState([]);
+
+  useEffect(() => {
+    const getWordList = async () => {
+      const data = await fetchWordList();
+      setWordList(data);
+    };
+
+    getWordList();
+  }, []);
```

Comment:
```suggestion
  useEffect(async () => {
    const data = await fetchWordList();
    setWordList(data);
  }, []);
```
Will this not work? Just a little shorter

Reply:
Not really, we want the async function inside UseEffect 
https://devtrium.com/posts/async-functions-useeffect